src/content/Kentico.Kontent.Boilerplate/Caching/CacheHelper.cs

```diff
@@ -174,7 +174,7 @@ public static IEnumerable<string> GetItemJsonDependencies(JObject response)
 
             return dependencies.Count > MAX_DEPENDENCY_ITEMS
                 ? new[] { GetItemsDependencyKey() }
-                : dependencies.Distinct();
```

Comment:
why this change?

Reply:
Dependencies variable is of type HashSet so no need to call distinct.

Reply:
true :)